ls1intum/Artemis
# `Exam mode`: Fix missing tutor comments in the second correction round

### Checklist
#### General


#### Client

### Motivation and Context


Closes #5030: Missing annotations of second exam correction round.

### Description

Thanks to @Shirkanesi and @dfuchss: The results of the second round of corrections are not part of the first result, so we need to look at the last result.

### Steps for Testing

Prerequisites:
- 1 Instructor + Tutor
- 1 Students
- 1 Programming Exercise
- 1 Exam with second correction round enabled

1. Log in to Artemis
2. Create an Exam with a Programming Exercise
3. Assess the submission in both correction rounds
4. Verify that the tutor/instructor sees the correct comments for the the current correction round

### Review Progress





#### Code Review
#### Manual Tests
